### ISSUE TITLE
Duda Test E2 Lance Agiity #172

### ISSUE THREAD

**Issue by SebaValenzuela** [franciscogazitua/IIC2113-2024#170]
Hola, tengo una duda sobre este test, el enunciado dice que el orden para anunciar los cambios en las stats dice que es Atk, Spd, Def y por ultimo Res. Pero en en este test y en el de Bow Agility el output sale primero Spd y despues Atk. Esto esta bien??

Gracias.

**Comment by RodrigoToroIcarte** [comment]
Sí, está bien. Lo que pasa es que primero se muestran los bonus y luego los penalties. 

![Screenshot from 2024-09-15 22-48-37](https://github.com/user-attachments/assets/c55d417c-04be-4fce-824f-f43c5b85cbd1)


En este caso Sigurd tiene un bonus de spd y un penalty de Atk. Por eso aparece Spd antes de Atk.

```
Sigurd obtiene Spd+12
Sigurd obtiene Atk-6
```

**Comment by SebaValenzuela** [comment]
perfecto graciass
